packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -59,6 +59,8 @@ export interface IEmailProvider extends IProvider {
   channelType: ChannelTypeEnum.EMAIL;
 
   sendMessage(options: IEmailOptions): Promise<ISendMessageSuccessResponse>;
+
+  checkIntegration(options: IEmailOptions): Promise<ICheckIntegrationResponse>;
```

Comment:
Why are we passing the email options here? I would suggest that each provider will handle this logic internally, since some of them might actually have a test endpoint. Wdyt?